packages/ai/src/ai/modules/mcp/apps/src/dropper/index.html

```
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)